src/Commands/Org/Site/ListCommand.php

```diff
@@ -57,6 +62,9 @@ public function listSites($organization, $options = ['plan' => null, 'tag' => nu
         if (!is_null($tag = $options['tag'])) {
             $this->sites->filterByTag($tag);
         }
+        if (!is_null($tags = $options['tags'])) {
+            $this->sites->filterByTags($tags);
+        }
```

Comment:
`tag` and `tags` probably shouldn't be used together, unless it's used in an and/or combination? Maybe there is a use for it.